config/profile.go

```diff
@@ -156,25 +162,84 @@ func (p *Profile) SetRootPath(rootPath string) {
 		p.Copy.PasswordFile = fixPath(p.Copy.PasswordFile, expandEnv, absolutePrefix(rootPath))
 		p.Copy.RepositoryFile = fixPath(p.Copy.RepositoryFile, expandEnv, absolutePrefix(rootPath))
 	}
+
+	// Handle dynamic flags dealing with paths
+	pathFlags := []string{
+		"cacert",
+		"tls-client-cert",
+		"cache-dir",
+		"repository-file",
+		"password-file",
+		constants.ParameterPath,
```

Comment:
Parameter `path` is resolved here, but shouldn't.

Reply:
oohhh that was the trick, I was looking for a "path" string. Well it looks like I should have a rest today 😆